packages/node/src/utils/time.ts

```diff
@@ -0,0 +1,52 @@
+// HAR timing Data adapted from
+// https://github.com/exogen/node-fetch-har/blob/master/index.js
+
+import { HttpRequest } from '@envyjs/core';
+
+export type Slice = [number, number];
```

Comment:
I wonder if a better name would be `HRTime` or `HighResolutionTimestamp` or something similar, as `Slice` implies (to me) that the tuple represents a `[start, end]` when it actually represents `[second, nanoseconds]`.